test/functional/tests/cli/test_cli_help_and_version.py

```diff
@@ -98,3 +98,20 @@ def test_cli_help(shortcut):
                                   + (" -H" if shortcut else " --help"))
     check_stderr_msg(output, unrecognized_stderr)
     check_stdout_msg(output, unrecognized_stdout)
+
+
+@pytest.mark.parametrize("shortcut", [True, False])
+def test_cli_version(shortcut):
+    """
+    title: Test for 'version' command.
+    description: Test if version displays.
+    pass_criteria:
+      - Proper OCL's components names displays in table with its versions.
+    """
+    TestRun.LOGGER.info("Check OCL's version.")
+    output = casadm.print_version(shortcut=shortcut).stdout
+    TestRun.LOGGER.info(output)
+    if ("CAS Cache Kernel Module" not in output
```

Comment:
Just wondering - maybe we do want to check if version is displayed in expected format (e.g. "20.06.00.00000518")?

Reply:
Good point, I will prepare a change and push it when unified versioning becomes reality.